server/utils/articles/article-search.utils.js

```diff
@@ -0,0 +1,24 @@
+import Response from '../response.util';
+
+class SearchValidators {
+/**
+ * @param {object} req
+ * @param {object} res
+ * @param {any} next
+ * @returns {any} response
+ */
+  static async checkQueryParams(req, res, next) {
+    const { author, title, tag } = req.query;
+    if (!author && !title && !tag) return Response(res, 422, 'No search parameters inputted');
+    return next();
+  }
+
+  static async checkSpecialChars(req, res, next) {
+    const { author, title, tag } = req.query;
+    const checkRegex = /[`~!@#$%^&*()_|+\-=?;:'",.<>{}\\/]/;
```

Comment:
Please, what is the function of this regex expression?

Reply:
@davidokonji the regex expression is used to check for the presence of special characters, which in the long run prevents SQL injection.